modules/custom/social_queue_storage/src/Form/QueueStorageEntityForm.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Drupal\social_queue_storage\Form;
+
+use Drupal\Core\Entity\ContentEntityForm;
+use Drupal\Core\Form\FormStateInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Form controller for Queue storage entity edit forms.
+ *
+ * @ingroup social_queue_storage
+ */
+class QueueStorageEntityForm extends ContentEntityForm {
+
+  /**
+   * The current user account.
+   *
+   * @var \Drupal\Core\Session\AccountProxyInterface
+   */
+  protected $account;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    // Instantiates this form class.
+    $instance = parent::create($container);
+    $instance->account = $container->get('current_user');
+    return $instance;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    /* @var \Drupal\social_queue_storage\Entity\QueueStorageEntity $entity */
+    $form = parent::buildForm($form, $form_state);
+
+    return $form;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function save(array $form, FormStateInterface $form_state) {
+    $entity = $this->entity;
+
```

Comment:
some documentation would be helpfull here, what do we expect this status to contain considering we do a switch statement with only 1 case other than default. (For someone extending this could be nice)